### ISSUE TITLE
Aggregator loading video links

### ISSUE THREAD

**Issue by saumier** [culturecreates/footlight-aggregator#16]

**Comment by saumier** [comment]
@troughc I published this event on ToutCulture to see that the video is displayed on Minimal's site. It seem to be working.
https://toutculture.ca/event-detail/?event=64ba31763c19fa0064e944d2

Please close after testing in production.

**Comment by troughc** [comment]
@saumier what is the test you would like me to do? Am I to see if the video link comes through with the aggregator?
**Here is what I have done so far (my best guess at what to test was to see if the video link came across):**

I looked for an event from Salle Odyssee that remains untouched in the CMS (is draft) and has video content. I found this one: https://www.maisondelaculture.ca/Salle-Odyssee/spectacles/chanson/les-cowboys-fringants-les-antipodes.html
I confirmed that the video link is included in the console: https://console.footlight.io/events/footlight%3Amaisondelaculture-ca_chanson-les-cowboys-fringants-les-antipodes
I then checked the event in the CMS; the video link is not included on the event details page: https://cms.footlight.io/dashboard/6308ef4a7f771f00431d939a/events/add-event/64af64643c19fa0064e7724a
